climpred/graphics.py

```diff
@@ -105,6 +97,29 @@ def plot_bootstrapped_skill_over_leadyear(init_skill,
             https://doi.org/10/f8wkrs.
 
     """
+    p = (100 - sig) / 100  # 0.05
+    ci_low = p / 2  # 0.025
+    ci_high = 1 - p / 2  # .975
+    pers_sig = sig
+
+    init_skill = bootstrapped.sel(kind='init', results='skill')
+    init_ci = bootstrapped.sel(kind='init',
+                               results=[ci_low, ci_high
+                                        ]).rename({'results': 'quantile'})
```

Comment:
The indentation is a bit distracting; maybe:
```
init_ci = bootstrapped.sel(
    kind='init', results=[ci_low, ci_high]).rename({'results': 'quantile'})
```
Also, since all of these rename results to quantile, would it be possible to
`bootstrapped_rn = bootstrapped.rename({'results': 'quantile'})` and then just call that to reduce copy/paste? and then do the same with `results = [ci_low, ci_high]`

Reply:
my beautifier changes this again.
I use `yapf` as a formatter based on pep8, max line length 79 and `multi line output` as `hangoing grid grouped`. Can we maybe agree on some more style guides some time soon?

Reply:
Yeah that can be handled with something like https://github.com/bradyrx/climpred/issues/125. I don't know why atom goes for the hanging indent -- I definitely prefer one liner readability if possible. If there's a way for you to switch off the hanging grid grouped that might help here.

Reply:
I specified the hanging indent some time ago I figured out. Unconscious choice